### DISCUSSION TITLE
Basic Auth for webserver, but not for a specific subfolder

### DISCUSSION THREAD

**Discussion by rucksman** [nginx-proxy/nginx-proxy#2502] [General]
I am using nginx-proxy since many years without a problem. On some of my webserver I use Basic Auth as described in the docs. Now I have a special requirement: All access to a webserver should be blocked by Basic Auth, but not for a specific subfolder.

To be more precise: Basic Auth should be in effect for my.domain.tld, but everybody should have access to my.domain.tld/no-auth.

How can I achieve that? I'm sure it's easy, but at the moment I'm at a loss.

**Comment by buchdag** [comment]
Hi.

Selectively disabling basic auth for specific paths is unfortunately not supported at the moment. 😞
